manifests/child/apache.pp

```diff
@@ -18,6 +18,12 @@
     $directories = undef
   }
 
+  if $ssl_ca {
```

Comment:
Can't this block use ```pick()``` as well?

Reply:
Unfortunately not, as pick evaluates all the values and `$::pulp::ca_cert` is not defined in all cases I believe. Does that make sense or am I missing something?

Reply:
Oh yes, the ```pulp::child::apache``` class is weird in its inheritance and something I haven't touched so far. You might be right on this one.